Add `History of osu!/2010` (#10622)
ppy/osu-wiki: wiki/History_of_osu!/2010/en.md

> # History of osu! 2010

Reviewer: while that's a start (and a good one), I'm sure there's still a lot to go through

```suggestion
---
stub: true
---

# History of osu! 2010
```

Author: Out of curiousity, do you have anything specific in mind that is not included here?

Reviewer: not really, at least not at the moment, just that the article feels too slim -- surely there has been a lot going on in 2010

## January

> On the [Wai-con](https://en.wikipedia.org/wiki/Wai-Con) anime convention in Perth, osu! had an arcade booth which featured a fully self-contained arcade machine running a touch-based version of osu! on it called osu!arcade. Although playing too much on it could lead to sore fingers as some people had criticised, the prototype containing custom-made hardware and a built-in touchscreen was well-received by the community.[^wai-con]

Reviewer: this reads like it may benefit from simplification
```suggestion
The [Wai-con](https://en.wikipedia.org/wiki/Wai-Con) anime convention in Perth had osu!arcade, a fully self-contained arcade machine running a touch-based version of osu!. Although playing too much on it could lead to sore fingers as some people had criticised, the prototype containing custom-made hardware and a built-in touchscreen was well-received by the community.[^wai-con]
```

> ![A touchscreen displaying osu!'s main menu](img/osu!arcade.jpg "The osu!arcade machine at 2010's Wai-con")

Reviewer: I think so -- the only problem is that `## February` below will likely be pushed down, but we'll need to check it in the preview mode to say for sure
```suggestion
::: Infobox
![A touchscreen displaying osu!'s main menu](img/osu!arcade.jpg "The osu!arcade machine at 2010's Wai-con")
:::
```

## February

> A new osu! update made small improvements in the overall user interface and added the ability to move all storyboard events by an arbitrary time value, allowing for more fine-tuned storyboard animations. In addition to this, usernames became clickable in chat logs.[^stable-b1460]

Reviewer: while these are helpful UX improvements, they don't sound significant at all (at least to me)

Reviewer: what I meant by this is that if an update has brought "small improvements" and did something mild to the storyboarding engine, it doesn't read as something worth bringing up at all, hence the comment

## March

> Through February and March, [peppy](https://osu.ppy.sh/users/2) organised an art competition about drawing banners for in-game challenges. After a [community voting](https://osu.ppy.sh/community/forums/topics/27112?n=1) concluded, the osu! team picked the final selection. The winner's art being officially used as well as one year of [osu!supporter](/wiki/osu!supporter) or physical goods being awarded were in prospect for the competition's finalist.[^art-challenge]

Reviewer: ```suggestion
Through February to March, [peppy](https://osu.ppy.sh/users/2) organised an art competition dedicated to banners for in-game challenges, with the prize being the art's official recognition and a year of [osu!supporter](/wiki/osu!supporter) or physical goods. After a [community voting](https://osu.ppy.sh/community/forums/topics/27112?n=1) round, the osu! team made the final choice.[^art-challenge]
```

> On March 8, an osu! update brought mainly performance and audio improvements to the game.[^stable-b1485]

Reviewer: to me, "mainly" sounds as the update itself was a regular one -- was it?
```suggestion
On March 8, an osu! update brought considerable performance and audio improvements to the game.[^stable-b1485]
```

Author: I guess you could say that.

## April

> Due to infrastructure changes, the [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat) was moved to the address `irc.ppy.sh`.[^irc]

Reviewer: ```suggestion
Due to infrastructure changes, the [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat) server was moved to the address `irc.ppy.sh`.[^irc]
```

This month, peppy introduced a new star rating difficulty algorithm in an osu! update.[^stable-b1596]

## August

> Since August 2, users became able to change their username once. To change their name, they had to have received an osu!supporter tag at least once beforehand, either through purchase or as a gift.[^name-change]

Reviewer: that's a question for @peppy

> Since August 2, users became able to change their username once. To change their name, they had to have received an osu!supporter tag at least once beforehand, either through purchase or as a gift.[^name-change]

Reviewer: advanced tenses are one of my weaknesses, but "users became able" sounds wrong to me really, so a tiny rewrite is in order
```suggestion
On August 2, all users had been granted a single username change. To change username, they had to had received an osu!supporter tag at least once, either through purchase or as a gift.[^name-change]
```

Reviewer 2: "they had to had received" is not correct i'm pretty sure

```suggestion
On August 2, all users were granted a single username change. To change username, they had to have received an osu!supporter tag at least once, either through purchase or as a gift.[^name-change]
```

Reviewer: thanks

> Since August 2, users became able to change their username once. To change their name, they had to have received an osu!supporter tag at least once beforehand, either through purchase or as a gift.[^name-change]

Reviewer 3: I presume that this is correct, and is when name changes first became available? That's what it's saying anyway.

> [Bancho](/wiki/BanchoBot) received the [`!report`](/wiki/BanchoBot#report) command, allowing people to notify chat moderators of users that showed unsolicited behaviour (e.g. spamming in chat).[^bancho-report]

Reviewer: ```suggestion
The [`!report`](/wiki/BanchoBot#report) chat command was added to [Bancho](/wiki/BanchoBot), which allowed people to notify chat moderators about users misbehaving in the chat.[^bancho-report]
```

[nardii](https://osu.ppy.sh/users/1017) gave peppy access to their [Twitter](https://twitter.com) account [@osugame](https://twitter.com/osugame) which was used for general osu! news from then on.[^twitter-osugame]

## September

> On September 2, osu! received several mapping updates. Among those were a display for showing how efficiently storyboard and background elements are drawn on the screen, known as [SB load](/wiki/Client/Beatmap_editor/SB_load). Additionally, users became able to set the speed of [approach circles](/wiki/Gameplay/Hit_object/Approach_circle) regardless of [overall difficulty](/wiki/Beatmap/Overall_difficulty). Slider speeds between 2x and 1/2x became able to be adjusted to custom values.[^stable-b1650]

Reviewer: ```suggestion
On September 2, osu! received several mapping updates, including:[^stable-b1650]

- The [SB load](/wiki/Client/Beatmap_editor/SB_load) indicator, reflecting drawing efficiency of storyboard and background elements.
- The introduction of [Approach Rate](/wiki/Beatmap/Approach_rate) for configuring the speed of [approach circles](/wiki/Gameplay/Hit_object/Approach_circle) independently of [Overall Difficulty](/wiki/Beatmap/Overall_difficulty).
- Custom [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) multipliers ranging from 0.5x to 2x.
```

60 [difficulties](/wiki/Beatmap/Difficulty) were affected by a Taiko bug causing drum-rolls to be incorrectly displayed and thus, all scores on them were deleted.[^taiko-reset]

> ## October

Author: I'm not sure how detailed the `Flashlight` mod fiasko through October and November should be mentioned here. To reiterate what happened, take a look at these threads:

**October:**

- [flashlight mod deactivated](https://osu.ppy.sh/community/forums/topics/38692?n=1)
- [flashlight is back](https://osu.ppy.sh/community/forums/topics/38760?n=1)

**November:**

- [flashlight disabled again](https://osu.ppy.sh/community/forums/topics/41039?n=1)
- [flashlight is back (again)](https://osu.ppy.sh/community/forums/topics/41519?n=1)

Reviewer: I like the idea, but sadly I don't recall any similar osu!-related issues having being posted on the wiki

Author: The `Flashlight` mod page already mentions that the mod had to be deactivated due to those bugs.

Also, there is the ongoing discussion of what to do with these history sections spread around the wiki.

I guess we can leave it be for now and just have this comment as reference for later.

Reviewer 3: I think it would be nice to document it.

> Members of the [Mapping Assistance Team](/wiki/People/Mapping_Assistance_Team) (MAT) became able to [bubble](/wiki/Modding/Bubble) beatmaps to indicate that they are of good quality.[^mat-bubble]

Reviewer: ```suggestion
Members of the [Mapping Assistance Team](/wiki/People/Mapping_Assistance_Team) (MAT) gained the ability to [bubble](/wiki/Modding/Bubble) beatmaps deemed to be of good quality.[^mat-bubble]
```

## April

Due to infrastructure changes, the [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat) server was moved to the address `irc.ppy.sh`.[^irc]

This month, peppy introduced a new star rating difficulty algorithm in an osu! update.[^stable-b1596]

## August

On August 2, all users were granted a single username change. To change username, they had to have received an osu!supporter tag at least once, either through purchase or as a gift.[^name-change]

The [`!report`](/wiki/BanchoBot#report) chat command was added to [Bancho](/wiki/BanchoBot), which allowed people to notify chat moderators about users misbehaving in the chat.[^bancho-report]

[nardii](https://osu.ppy.sh/users/1017) gave peppy access to their [Twitter](https://twitter.com) account [@osugame](https://twitter.com/osugame) which was used for general osu! news from then on.[^twitter-osugame]

## September

On September 2, osu! received several mapping updates, including:[^stable-b1650]

- The [SB load](/wiki/Client/Beatmap_editor/SB_load) indicator, reflecting drawing efficiency of storyboard and background elements.

> - The introduction of [Approach Rate](/wiki/Beatmap/Approach_rate) for configuring the speed of [approach circles](/wiki/Gameplay/Hit_object/Approach_circle) independently of [Overall Difficulty](/wiki/Beatmap/Overall_difficulty).

Author: The [ASC § Capitalisation](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#capitalisation) give a similar example 🤔

> - The introduction of [Approach Rate](/wiki/Beatmap/Approach_rate) for configuring the speed of [approach circles](/wiki/Gameplay/Hit_object/Approach_circle) independently of [Overall Difficulty](/wiki/Beatmap/Overall_difficulty).

Reviewer 2: ah, that's outdated. we don't capitalise AR, CS, HP, OD when written out

- Custom [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) multipliers ranging from 0.5x to 2x.

> 60 [difficulties](/wiki/Beatmap/Difficulty) were affected by a Taiko bug causing drum-rolls to be incorrectly displayed and thus, all scores on them were deleted.[^taiko-reset]

Author: The reference of `Taiko` here does count as a historic mention, doesn't it? This writing style is also used in the forum threads.

Out of curiousity, when was the writing style changed to `osu!taiko` and `osu!catch`, and should this be reflected in the wiki, like described in the [ASC § osu!](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#osu!)?

> 60 [difficulties](/wiki/Beatmap/Difficulty) were affected by a Taiko bug causing drum-rolls to be incorrectly displayed and thus, all scores on them were deleted.[^taiko-reset]

Reviewer 2: as in the game mode name was "Taiko" back then? if that's the case, it can probably stay as is

may want to explain somehow that it's the old name though
